### DISCUSSION TITLE
Will plc-simulator consider open source?

### DISCUSSION THREAD

**Discussion by blueYangAAYY** [codingplc/plc-simulator#5] [Ideas]
I desperately need this feature, will you open source it?  Thank you and look forward to your reply.

**Comment by dagryl** [comment]
Why would you like me to open source it. How would it benefit the project?

**Comment by blueYangAAYY** [comment]
I like your project

**Reply by dagryl** [reply]
That doesn't answer my questions. :)
I would consider opening the source code if it benefited the project.

**Reply by blueYangAAYY** [reply]
Looking forward to the day, thank you for your reply, your project has been very helpful to our automation students

**Comment by dagryl** [comment]
Good news! [PLC Simulator Online Goes Open Source
](https://www.codingplc.com/blog/plc-simulator-online-goes-open-source)
